openapi-model/src/main/scala/sttp/apispec/openapi/validation/OpenAPICompatibilityIssue.scala

```diff
@@ -0,0 +1,137 @@
+package sttp.apispec.openapi.validation
+
+import sttp.apispec.openapi.ResponsesKey
+import sttp.apispec.validation.SchemaCompatibilityIssue
+
+sealed abstract class OpenAPICompatibilityIssue {
+  def description: String
+
+  override def toString: String = description
+  protected def issuesRepr(issues: List[OpenAPICompatibilityIssue]): String =
+    issues.iterator
+      .map(i => s"- ${i.description.replace("\n", "\n  ")}") // indent
+      .mkString("\n")
+}
+
+sealed abstract class SubOpenAPICompatibilityIssue extends OpenAPICompatibilityIssue {
```

Comment:
What is "sub-OpenAPI"? Or is it an OpenAPI "sub-issue"?

Note: `SubschemaCompatibilityIssue` makes sense because there is a notion of "subschema" - it is the vocabulary used by JSON Schema specification. However, I don't believe there is something we could call a "sub-OpenAPI".